src/sgame/sg_bot_parse.cpp

```diff
@@ -1169,7 +1169,7 @@ static AIGenericNode_t *ReadActionNode( pc_token_list **tokenlist )
 	BotInitNode( ACTION_NODE, action->run, &node );
 
 	// allow dropping of parenthesis if we don't require any parameters
-	if ( action->minparams == 0 && parenBegin->token.string[0] != '(' )
+	if ( action->minparams == 0 && parenBegin != nullptr && parenBegin->token.string[0] != '(' )
```

Comment:
I guess this would prevent a crash but it would still improperly reject a behavior tree consisting of a single action with no arguments.

Reply:
Of course it does. I am just changing from "crash" to "reject".

Reply:
If you prefer the crash, you can still have it with
```
condition alertedToEnemy
```